### PR TITLE
Generator functions now accept edgelists as graphs

### PR DESCRIPTION
**Pull request by arcondello**

### CODE REVIEW

File: tests/test_decorators.py [hunk 1]
```diff
@@ -147,6 +147,50 @@ def f(G):
...
+        @graph_argument('G')
+        def f(G):
+            return G
+
+        edgelist = [(0, 1), (2, 3), (0, 2)]
+
+        nodes, edges = f(edgelist)
+
+        for edge in edges:
+            self.assertIn(edge, edgelist)
```
Comment (m3ller) [inline tests/test_decorators.py]: Shouldn't you also verify that when using simply the `edgelist` that the nodes are generated correctly as well?

File: tests/test_decorators.py [hunk 2]
```diff
@@ -147,6 +147,50 @@ def f(G):
...
+        @graph_argument('G')
+        def f(G):
+            return G
+
+        edgelist = [(0, 1), (2, 3)]
+
+        nodes, edges = f(edgelist)
+
+        for edge in edges:
+            self.assertIn(edge, edgelist)
```
Comment (m3ller) [inline tests/test_decorators.py]: ditto

File: tests/test_decorators.py (hunk 1 above)
Comment (arcondello) [reply tests/test_decorators.py]: Oops, good catch! should be `self.assertIn(edge, edges)`

Comment (arcondello) [reply tests/test_decorators.py]: Yes probably, will add

**Pull request merged**